inc/wp-components/classes/class-gutenberg-content.php

```diff
@@ -118,6 +136,10 @@ private function convert_block_to_component( $blocks, $current_block ) : array {
 			$block
 		);
 
+		if ( $this->components_callback ) {
```

Comment:
```suggestion
		if ( is_callable( $this->components_callback ) ) {
```

Reply:
Probably not super necessary since it's private and you have the helper locking it down, but can't hurt.